code/game/objects/items/emags.dm

```diff
@@ -22,6 +22,56 @@
 		user.visible_message(span_notice("[user] shows you: [icon2html(src, viewers(user))] [name]."), span_notice("You show [src]."))
 	add_fingerprint(user)
 
+/obj/item/card/emag/emag_act(mob/user, obj/item/card/emag/emag_card)
+	if(isnull(user) || !istype(emag_card))
+		return FALSE
+	var/emag_count = 0
+	for(var/obj/item/card/emag/emag in contents + emag_card.contents)
+		emag_count++
+	if(emag_count >= 6) // 1 uplink's worth is the limit
+		to_chat(user, span_warning("Nope, lesson learned. No more."))
+		return FALSE
+	if(emag_card.loc != loc) // Both have to be in your hand (or TK shenanigans)
+		return FALSE
+	if(!user.transferItemToLoc(emag_card, src))
+		return FALSE
+
+	user.visible_message(
+		span_notice("[user] holds [emag_card] to [src], getting the two cards stuck together!"),
+		span_notice("As you hold [emag_card] to [src], [emag_card.p_their()] magnets attract to one another, \
+			and [emag_card.p_they()] become stuck together!"),
+		visible_message_flags = ALWAYS_SHOW_SELF_MESSAGE,
+	)
+	playsound(src, 'sound/effects/bang.ogg', 33, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+	addtimer(CALLBACK(src, PROC_REF(contemplation_period), user), 2 SECONDS, TIMER_DELETE_ME)
+	emag_card.pixel_x = pixel_x + 4
+	emag_card.pixel_y = pixel_y + 4
+	emag_card.layer = layer - 0.01
+	emag_card.vis_flags |= VIS_INHERIT_ID|VIS_INHERIT_PLANE
+	vis_contents += emag_card
+	name += "-[emag_card.name]"
+	if(desc == initial(desc))
```

Comment:
honestly, skip this imo.

"It's a card with a magnetic strip attached to some circuitry. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly. There seems to be another emag stuck to it...pretty soundly." is peak.

Reply:
I did have the desc stack forever but it stacked like 
```
"It's a card with a magnetic strip attached to some circuitry. There seems to be another emag stuck to it...pretty soundly. It's a card with a magnetic strip attached to some circuitry. There seems to be another emag stuck to it...pretty soundly."
```
rather than the funny way 

I guess I can snowflake it though